Editor/CoreAPI/ScenarioParameterKeys.cs

```diff
@@ -9,7 +9,7 @@ public static class ScenarioParameterKeys
         public const string LaunchPath = "LaunchPathParameter";
         public const string LaunchParameters = "LaunchParameters";
         public const string BuildOperatingSystem = "BuildOperatingSystem";
-        public const string FleetName = "FleetName";
+        public const string FleetName = "AnywhereFleetName";
```

Comment:
Hmm, Aren't the scenarios supposed to correspond to Managed fleets and not Anywhere fleets?

Reply:
Incorrectly changed